qualtran/qref_interop/bartiq_demo.ipynb

```diff
@@ -101,60 +657,84 @@
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "id": "56beb66e-0989-4cf6-920b-bbc51cae559d",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Qualtran Bloq T Complexity: \n",
+      "TComplexity(t=307, clifford=1770, rotations=2)\n",
+      "\n",
+      "QREF Resources:\n",
+      "{'clifford': <Resource name=\"clifford\" value=\"1770\">,\n",
+      " 'rotations': <Resource name=\"rotations\" value=\"2\">,\n",
+      " 't': <Resource name=\"t\" value=\"307\">}\n"
+     ]
+    }
+   ],
    "source": [
     "from bartiq.integrations.qref import qref_to_bartiq\n",
     "from bartiq import compile_routine\n",
     "\n",
+    "\n",
+    "# Clear the resources from the top-level\n",
     "qref_definition.program.resources =  [] \n",
+    "\n",
     "bartiq_routine = qref_to_bartiq(qref_definition)\n",
     "compiled_routine = compile_routine(bartiq_routine)\n",
+    "\n",
+    "print(\"Qualtran Bloq T Complexity: \")\n",
+    "pprint(bloq.t_complexity())\n",
+    "print(\"\\nQREF Resources:\")\n",
     "pprint(compiled_routine.resources)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "2db6c501-5756-4356-ae6f-a6f412155e68",
+   "id": "31b2ceb7",
    "metadata": {},
    "source": [
-    "As we can see, Bartiq correctly reconstructed the costs from the subroutines."
+    "## Writing a Custom Uniform Superposition (USP) Implementation\n",
+    "\n",
+    "While the example above demonstrated the integration, the information derived by Bartiq was the same as the information from Qualtran. Therefore, let's try other scenarios highlighting use cases where Bartiq can provide unique information.\n"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "27483d83-d008-499c-9322-c2953f6db573",
+   "id": "91aa9d2b",
    "metadata": {},
    "source": [
-    "## Writing custom USP implementation\n",
-    "\n",
-    "So far so good, but we used Bartiq to derive the same information we already had in Qualtran.\n",
-    "\n",
-    "There are two limitations of Qualtran implementation, with which Bartiq can help us, let's see what these are.\n",
+    "### Numeric Implementation\n",
     "\n",
-    "### Numeric implementation\n",
-    "\n",
-    "The example above is purely numeric. We can get the symbolic cost of the whole `StatePreparationAliasSampling` (see snippet below) but we can't decompose it using `decompose_bloq()`.\n",
+    "The example above is purely numeric. If symbolic expressions are used, we are able to produce resources and decompose the \"top-level\" bloq, `StatePrepationAliasSampling` (see snippet below). However, subsequent bloqs, like `StatePreparationAliasSampling`, cannot be decomposed using `decompose_bloq()`.\n",
```

Comment:
```suggestion
    "The example above is purely numeric. If symbolic expressions are used, we are able to produce resources and decompose the \"top-level\" bloq, `StatePreparationAliasSampling` (see snippet below). However, subsequent bloqs, like `PrepareUniformSuperposition`, cannot be further decomposed using `decompose_bloq()` This is most likely temporary and as Qualtran improves, this might cease being a problem.\n",
```